src/test/common/fdc3.user-channels.ts

```diff
@@ -282,6 +282,24 @@ export function createUserChannelTests(cc: ChannelControl<any, any, any>, docume
       cc.unsubscribeListeners([listener, listener2]);
     });
 
+
+    const scTestId9 = "(" + prefix + "UCFilteredUsageNoJoin) Should not receive context when A doesn't join a user channel before app B broadcasts the listened type to that channel";
+    it(scTestId9, async () => {
+      const errorMessage = `\r\nSteps to reproduce:\r\n- App A adds context listener of type fdc3.instrument\r\n- App A unsubscribes the listener\r\n- App B joins channel 1\r\n- App B broadcasts context of type fdc3.instrument${documentation}`;
```

Comment:
Definition a reproduction steps don't match. Definition says you don't join, replication steps you join and leave. Implementation has you not not joining.

Whats being tested? Not being joined or joining and leaving?  Presumably it is ensured that we are not joined to a channel from a previous test (so its almost certainly joining and leaving whatever you do).